Doc/c-api/long.rst

```diff
@@ -144,13 +143,15 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. deprecated:: 3.8
       Using :meth:`__int__` is deprecated.
 
+   .. versionchanged:: 3.10
+      This function will no longer use :meth:`__int__`.
```

Comment:
We usually also remove corresponding `deprecated` directives when the deprecated feature has been removed. So I suggest to remove `deprecated` few lines above.

Reply:
Done!